src/components/searchWidget.js

```diff
@@ -29,6 +28,25 @@ export default class searchWidget extends Component {
     }
     
     fetchRepos(keyword, orgs){
+        //creates component with template return for unit testing
+        if(keyword == "bypassFetch"){
+            return [
```

Comment:
seems to me a better approach is to have some sort of mock to return this test result array from the githubApiInterface.getJSON call. So we actually cover the same code path.

Reply:
That sounds like a good approach